docs/source/concepts/DynamicalStructureFactorFromAbInitio.rst

```diff
@@ -0,0 +1,163 @@
+order transitions form the following  set: :math:`\lbrace \omega_1 + \omega_1, \omega_1 + \omega_2, \omega_1 + \omega_3, \ldots,  \omega_n + \omega_n \rbrace`.
+Cardinality of this set is :math:`n^2`. In practice one can reduce this number by taking into consideration realistic energy  window
+and neglecting those :math:`\omega_{ij}=\omega_i + \omega_j` for which :math:`S(Q, \omega_i)` is negligible.
+Within harmonic approximation each phonon is treated as independent harmonic quantum oscillator.  The formula for :math:`S(Q, \omega_{ik})` is as follows [4]:
+
+:math:`S^j(Q, \omega_{ik}) = \frac{Q^4}{15  C}\left( TrB^j_{\omega_i}TrB^j_{\omega_k} + B^j_{\omega_i}:B^j_{\omega_k} + B^j_{\omega_k}:B^j_{\omega_i} \right) exp\left(-Q^2(\omega_{ik}) \beta^j coth^2\left(\frac{\hbar \omega_{ik}}{2 k_B T} \right) \right)\sigma^j`
+
+where:
+
+:math:`\beta^j = A^j / 3`
+
+C -- is a constant,  C=2 if i=j and C=1 otherwise
+
+Similarly one can define contribution for the third quantum order event (0 -> 3, simultaneous  0 -> 1, 0 -> 1', 0 -> 1'' for different oscilators ,etc.. ) [4]:
+
+:math:`S^j(Q, \omega_{ikl}) = \frac{9Q^6}{543}\left( TrB^j_{\omega_i} TrB^j_{\omega_k} TrB^j_{\omega_l}  \right)  exp\left(-Q^2) \beta^j coth^2\left(\frac{\hbar \omega_{ikl}}{2 k_B T}\right) \right)\sigma^j`
+
+Usually in order to reconstruct experimental spectrum it is enough to give contribution up to fourth order (0 -> 4 , simultaneous 0 -> 1, 0 -> 1', 0 -> 1'', 0- > 1''' for different oscilators, etc.)  [4]:
+
+:math:`S^j(Q, \omega_{iklm}) = \frac{27Q^8}{9850}\left( TrB^j_{\omega_i} TrB^j_{\omega_k} TrB^j_{\omega_l}TrB^j_{\omega_m}  \right) exp\left(-Q^2 \beta^j coth^2\left(\frac{\hbar \omega_{iklm}}{2 k_B T}\right) \right)\sigma^j`
+
+In the similar way as for the second quantum order event one can reduce number of energy transitions by taking into account considered energy window and neglecting small S.
+
+After evaluating equations above one obtains discrete S for each quantum order and for each atom. In order to compare such a spectrum (:math:`S_{theory}`) one has to convolve it with experimental resolution:
+
+:math:`S_{theory}^{nj}(Q, \omega) = S_{discrete}^{nj}(Q, \omega) * f(\omega)`
+
+where:
+
+:math:`j` -- indicates j-th atoms
+
+:math:`n` -- indicates n-order event
+
+:math:`f(\omega)` -- resolution function
+
+For `Tosca <http://www.isis.stfc.ac.uk/instruments/tosca/tosca4715.html>`_  and Tosca-like instruments :math:`f(\omega)` has the following form:
+
+:math:`f(\omega)=1.0 / \sqrt{\sigma(\omega)  \pi}  \exp(-(\omega)^2  / \sigma(\omega))`
+
+where:
+
+:math:`\sigma(\omega) = A  \omega^2  + B  \omega + C`
+
+with A, B, C as constants.
+
+Moreover, in case of Tosca and Tosca-like instruments length of momentum transfer depends on frequency (*indirect geometry spectrometer*).
+The formula for Q is as follows:
+
+:math:`Q(\omega)=k2_i(\omega) + k2_f - 2  \sqrt{k2_i(\omega)  k2_f} angle`
```

Comment:
Formula is wrong. it should be `Q^2(\omega)=k^2_i(\omega) + k^2_f - 2  k_i(\omega)  k_f \cos \theta`. Then in the next lines replace `k2` with `k^2`. And you should never name the cosine as `angle`. `\theta` is the scattering angle